docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc

```diff
@@ -124,6 +124,43 @@ Promise.all([neoSchema.getSchema(), ogm.init()]).then(([schema]) => {
 
 It's important to note the JWT secret being passed into the `Neo4jGraphQL` constructor in this example.
 
+Additionally, an example implementation for the util functions `createJWT` and `comparePassword` is provided in this code snippet:
+[source, javascript, indent=0]
+----
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
```

Comment:
Perhaps we can use import here instead, now?

Reply:
I first had it as import statements but most other code snippets in this subfolder use `require` so I went with the majority. Bit of a mixed bag

Reply:
OK, might be a thing to do as a sweeping change later then.

Reply:
I would argue so. First and foremost to keep consistency, especially within a single file/page.